pyproject.toml

```diff
@@ -133,6 +133,7 @@ social-auth-core = {version = "~4.3.0", optional = true, extras = ["openidconnec
 social-auth-app-django = "~5.0.0"
 # Rendering of SVG images (for rack elevations, etc.)
 svgwrite = "~1.4.2"
+djangorestframework-bulk = "^0.2.1"
```

Comment:
Can we do any better than pulling in a library that was last updated in **2015**?

Reply:
This project is [MIT-licensed](https://github.com/miki725/django-rest-framework-bulk/blob/master/LICENSE.rst) so we could easily cargo-cult [`rest_framework_bulk/mixins.py`](https://github.com/miki725/django-rest-framework-bulk/blob/master/rest_framework_bulk/drf3/mixins.py) into our core, give proper credit in our [`NOTICE`](https://github.com/nautobot/nautobot/blob/develop/NOTICE) file, and remove this dependency.